packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -19,17 +21,39 @@ const BotBuilder = ({ app, active_tab }: TBotBuilder) => {
     }, []);
 
     return (
-        <div className={classNames('bot-builder', { 'bot-builder--active': active_tab === 1 })}>
-            <div
-                id='scratch_div'
-                style={{
-                    width: 'calc(100vw - 3.2rem)',
-                    height: 'var(--bot-content-height)',
-                }}
-            >
-                <WorkspaceWrapper />
+        <>
+            <div className={classNames('bot-builder', { 'bot-builder--active': active_tab === 1 })}>
+                <div
+                    id='scratch_div'
+                    style={{
+                        width: 'calc(100vw - 3.2rem)',
+                        height: 'var(--bot-content-height)',
+                    }}
+                >
+                    <WorkspaceWrapper />
+                    {active_tab === 1 && (
+                        <ReactJoyride
+                            steps={BOT_BUILDER_TOUR}
+                            continuous={true}
+                            showProgress={true}
+                            styles={{
+                                options: {
+                                    arrowColor: 'var(--text-general)',
+                                    backgroundColor: 'var(--text-general)',
+                                    overlayColor: 'rgba(0, 0, 0, 0.5)',
+                                    primaryColor: '#FF444F',
```

Comment:
Do we have this color or can we add this on in our color variables?

Reply:
We have this. Changed to our color variable.